src/material/button/button.md

```diff
@@ -33,10 +33,22 @@ screen-readers, which will read the text character-by-character. We leave the de
 approach this to the consuming app.
 
 ### Accessibility
-Angular Material uses native `<button>` and `<a>` elements to ensure an accessible experience by
-default. The `<button>` element should be used for any interaction that _performs an action on the
-current page_. The `<a>` element should be used for any interaction that _navigates to another
-view_.
+Angular Material uses native <button> and <a> elements. Both of the two look the same when applied Material Design Styling, but have different semantic meaning for assitive technology. To ensure an accessible experience, choose the element which is most approprite for the purpose of the interaction.
```

Comment:
> but have different semantic meaning for assistive technology

This phrasing makes it seem like they _only_ have different semantic meaning for AT, whereas they in fact have fundamentally different purposes for all users, not only those that use AT. For example, mouse users can right click links to open them in a new window, and cannot for buttons. You may consider rephrasing this, but feel free to ack.

Reply:
+1 to it not being specific to assistive technoogy, I don't think there was anything wrong with the original sentence here. The goal here is to expand the content where necessary to fill in gaps rather than to completely rewrite everything.

Reply:
Please wrap markdown at 100 columns

Reply:
HTML tags should be wrapped in code ticks